### ISSUE TITLE
Tests fail in build -- jena cannot load some files

### ISSUE THREAD

**Issue by barthasanyi** [SPYDERISK/system-modeller#212]
### TL;DR: 

I run a `gradlew build` and 32 tests failed. The generated html output files are here:
[results.tar.gz](https://github.com/user-attachments/files/17525368/results.tar.gz)

I am on the `dev` branch with last commit `e6d7b1bd6775798a65857e0103afb8968d1ccc21`

### More information:

I am trying to set up a development environment following this guide:
[https://github.com/Spyderisk/system-modeller/blob/dev/docs/development.md](https://github.com/Spyderisk/system-modeller/blob/dev/docs/development.md)

I downloaded and copied the domain model domain-network-6a6-1-2.zip as instructed to the knowledgebases folder.
From inside docker, I successfully run the 
`/code# ./gradlew bootTest`
command, and I was able to log in to the system modeler webapp from the host browser. 

I wanted to run a development environment, so I stopped the bootTest version, and started a build:

`/code# ./gradlew build'

but 32 tests failed.

**Comment by kenmeacham** [comment]
I'll take a look today..

**Comment by kenmeacham** [comment]
Firstly, just an observation that you would only need to run all the unit tests (via `gradlew build`) if you wanted to test some Java code changes. If you simply wanted to launch system modeller, this can be done via:

`/code# ./gradlew assemble bootTest`

The above command builds and packages everything that you need for running system modeller, without running the unit tests. Much quicker! (@danshearer - perhaps we should fix the docs here?)

The `gradlew build` does however highlight some problems with your local setup. Looking at the logs from the tests, they all fail with a similar problem, e.g.

`Caused by: org.apache.jena.riot.RiotException: [line: 1, col: 1 ] Content is not allowed in prolog.`

This indicates that an RDF file that is being used for the test is corrupted (i.e. not RDF format). We have seen this issue before, when the code has been checked out, but without first installing git-lfs. Did you do this step? Otherwise, the RDF file will look something like this, instead of expected RDF content:

```
version https://git-lfs.github.com/spec/v1
oid sha256:923ebcb8db25f956adcd1f0cad33124779daec5e37d1fac3d8a6b735b0572510
size 3745
```

You can check this by having a look at any RDF files that have been checked out in the source tree, e.g. the following command would show the content above, if git-lfs is NOT installed:
`head src/test/resources/semanticstore/bm5.rdf`

If this is the case, you need to install git-lfs (as per the docs). You should then check out the code cleanly and continue as before.

Let me know how you get on. Any questions, just ask.

**Comment by barthasanyi** [comment]
Indeed the RDF files do not have the expected content:

```
sanka@rocinante:~/work/Spyderisk/system-modeller$ head src/test/resources/semanticstore/bm5.rdf
version https://git-lfs.github.com/spec/v1
oid sha256:923ebcb8db25f956adcd1f0cad33124779daec5e37d1fac3d8a6b735b0572510
size 3745
```

But `git-lfs` is installed:
```
sanka@rocinante:~/work/Spyderisk/system-modeller$ git-lfs --version
git-lfs/3.5.0 (GitHub; linux amd64; go 1.23.2)
```

**Comment by barthasanyi** [comment]
It turns out, thanks @scp93ch , that I have not done `git lfs pull`.
This step is missing from 
[https://github.com/Spyderisk/system-modeller/blob/dev/docs/development.md](https://github.com/Spyderisk/system-modeller/blob/dev/docs/development.md)

Altough I guess I should have figured it out (since git-lfs is a requirement).

**Comment by barthasanyi** [comment]
The necessary step is added to the docs by commit 9ed4e8a
